django-backend/fecfiler/reports/models.py

```diff
@@ -137,6 +137,16 @@ def delete(self):
 
         """delete report-transaction links"""
         ReportTransaction.objects.filter(report=self).delete()
+
+        for field in self._meta.fields:
+            if (
+                isinstance(field, models.ForeignKey)
+                and field.remote_field.on_delete == models.CASCADE
+            ):
+                foreign_key = getattr(self, field.name)
+                if foreign_key:
+                    foreign_key.delete()
```

Comment:
<img width="586" alt="image" src="https://github.com/fecgov/fecfile-web-api/assets/97105825/3e7d1d63-c37d-4c38-be3d-e16a26c9783d">

this includes the committee account the report is owned by.

You could loop through TABLE_TO_FROM:

```
for form_key in self.TABLE_TO_FROM:
  form = getattr(self, form_key)
  if form:
     form.delete()
```

Reply:
Replaced with iterating through the table as suggested.